libvast/src/taxonomies.cpp

```diff
@@ -110,6 +110,87 @@ bool operator==(const concept_& lhs, const concept_& rhs) {
   return lhs.concepts == rhs.concepts && lhs.fields == rhs.fields;
 }
 
+caf::error convert(const data& d, models_type& out) {
+  if (const auto& c = caf::get_if<record>(&d)) {
+    auto n = c->find("name");
```

Comment:
I find the `field` suffix confusing, since `field` already has a meaning in this context. I think it would be most appropriate to rename it to `name_data` so the reader can easily tell it apart from the extracted `name`.

Reply:
Hm, I personally don't find it more intuitive. Let's keep it short then.